lib/perl/Genome/Qc/Tool/Picard.pm

```diff
@@ -20,9 +20,9 @@ sub get_metrics {
 
     my $file = $self->qc_metrics_file;
     my $gmt_class = $self->gmt_class;
-    my %metrics = $self->metrics;
     my $metric_results = $gmt_class->parse_file_into_metrics_hashref($file);
-    return $self->_get_metrics(\%metrics, $metric_results);
+    my $metric_header_as_key = $gmt_class->can('_metric_header_as_key') ? $gmt_class->_metric_header_as_key : undef;
+    return $self->_flatten_metrics_hash($metric_results, metric_header_as_key => $metric_header_as_key);
```

Comment:
The alternative is to remove READ_GROUP metric accumulation from the QC config.  Once we have a solution for representing read group metrics in our QC metrics, we could turn read group reporting back on.